apps/files/templates/files/viewer.html

```diff
@@ -130,6 +134,24 @@
         {{ _('Fetching file.') }}
         </p>
       </div>
+      {% if automated_signing %}
+        <div id="validation-options">
+          <div>
+            <label>
+              <input type="checkbox" id="signing-hide-unnecessary" checked="true">
+              {{ _('Hide messages not required for automated signing') }}
+            </label>
+          </div>
+          <div id="signing-hide-ignored-container"
+                {% if not (validation_data and validation_data.signing_ignored_summary) %}class="js-hidden"{% endif
+            %}>
+            <label>
+              <input type="checkbox" id="signing-hide-ignored" checked="true">
+              {{ _('Hide messages present in previous version and ignored') }}
```

Comment:
That sounds a bit confusing. Would "Hide previously ignored messages' be still accurate enough?